gdx/src/com/badlogic/gdx/utils/reflect/Method.java

```diff
@@ -146,4 +146,18 @@ public Annotation getDeclaredAnnotation (Class<? extends java.lang.annotation.An
 		return null;
 	}
 
+	/** @return true if the method is synthetic */
+	public boolean isSynthetic () {
+		return method.isSynthetic();
+	}
+
+	/** @return true if the method is a bridge method */
+	public boolean isBridge () {
+		return method.isBridge();
+	}
+
+	public int getModifiers () {
+		return method.getModifiers();
+	}
+
```

Comment:
What does this have to do with _AlignTest / BinaryHeapTest / SortTest_?

Reply:
@SimonIT I thought I was going crazy when I last looked for this and couldn't find it. It appears to be accidentally included from #7517 - you'll see these four users are working together and haven't mastered Git.

Reply:
Yeah... I thought I had déjà vu^^ The thing is, we will probably run into an unnecessary merge conflict if we merge both PRs